bazel/repository_locations.bzl

```diff
@@ -1077,16 +1077,17 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2021-07-13",
         cpe = "N/A",
     ),
-    emscripten_toolchain = dict(
+    emsdk = dict(
         project_name = "Emscripten SDK",
         project_desc = "Emscripten SDK (use by Wasm)",
         project_url = "https://github.com/emscripten-core/emsdk",
-        version = "3.1.7",
-        sha256 = "bcceced0b7cad2e08375adf74ef20fa431230abbae8766bdad268c43e34f8d03",
-        strip_prefix = "emsdk-{version}",
+        # v3.1.7 with Bazel fixes
+        version = "0ea8f8a8707070e9a7c83fbb4a3065683bcf1799",
+        sha256 = "1ca0ff918d476c55707bb99bc0452be28ac5fb8f22a9260a8aae8a38d1bc0e27",
+        strip_prefix = "emsdk-{version}/bazel",
         urls = ["https://github.com/emscripten-core/emsdk/archive/{version}.tar.gz"],
-        use_category = ["build"],
-        release_date = "2022-03-07",
+        use_category = ["test_only"],
```

Comment:
Does it mean we use a test version?

Reply:
It means this project is only compiled and used for tests (not dataplane, controlplane, etc.).
See: https://github.com/envoyproxy/envoy/blob/293965652ff8782143ce3be9c25a8109f522c125/api/bazel/external_deps.bzl#L45-L68

Reply:
Oh, I miss it, thanks!